Purpose: Hurd  is missing PATH_MAX, provide workaround.
Authour: Werner Fink
Fixes:   ?
Status:  Not send upstream

--- sysvinit.orig/src/killall5.c	(revision 1869)
+++ sysvinit/src/killall5.c	(working copy)
@@ -43,6 +43,7 @@
 #include <dirent.h>
 #include <errno.h>
 #include <getopt.h>
+#include <limits.h>
 #include <mntent.h>
 #include <stdarg.h>
 #include <stdio.h>
@@ -59,6 +60,14 @@
 
 char *Version = "@(#)killall5 2.86 31-Jul-2004 miquels@cistron.nl";
 
+#ifndef PATH_MAX
+#  ifdef MAXPATHLEN
+#    define PATH_MAX MAXPATHLEN
+#  else
+#    define PATH_MAX 2048
+#  endif
+#endif
+
 #define STATNAMELEN	15
 #define DO_NETFS 2
 #define DO_STAT 1
